sdk/identity/identity/CHANGELOG.md

```diff
@@ -40,7 +40,7 @@
 
 ### Breaking changes from 2.0.0-beta.1
 
-- Removed `VisualStudioCodeCredential`, as it requires us to list `keytar` as an optional dependency. `keytar` containes machine-code components that are difficult to build in certain environments, so this credential will be offered through a separate extension package in the future.
+- Removed `VisualStudioCodeCredential`, as it requires us to list `keytar` as an optional dependency. `keytar` contains machine-code components that are difficult to build in certain environments, so this credential will be offered through a separate extension package in the future.
```

Comment:
I think we should link to that separate extension package: https://www.npmjs.com/package/@azure/identity-vscode

Reply:
identity-vscode is mentioned above, in the New Features section of the 2.0.0-beta.4 (Unreleased). At the time of this change, we didn’t know the name of the package 🤔 this change I made here was to fix a typo. Should we still link to identity-vscode here even if the package wasn’t know at the time?

Reply:
Good point. Disregard my suggestion to add the package link.